package.json

```diff
@@ -46,7 +46,7 @@
     "sinon": "9.0.3",
     "testcafe": "1.8.6",
     "webpack": "4.44.1",
-    "webpack-cli": "3.3.9"
+    "webpack-cli": "^3.3.9"
```

Comment:
The dependencies should be locked to a specified version, so this caret should be removed.